tensorboard/plugins/interactive_inference/utils/inference_utils.py

```diff
@@ -829,7 +829,18 @@ def run_inference(examples, serving_bundle):
 
     values = []
     for pred in preds:
-      values.append(pred[preds_key])
+      # If the prediction dictionary contains only one key, use that key.
+      returned_keys = list(pred.keys())
+      if len(returned_keys) == 1:
+        key_to_use = returned_keys[0]
```

Comment:
This will pick whatever the model is outputting even in the case where predict_output_tensor is set. If the model is set up incorrectly and is not returning the tensor user wants, this may mislead the user into thinking they are actually using the tensor they specified in predict_output_tensor.

Reply:
Fixed